File: app/src/main/java/com/softaai/designpatterns/java/structural/bridge/RedCircle.java
package com.softaai.designpatterns.java.structural.bridge;

/**
 * Created by Amol Pawar on 03-04-2019.
 * softAai Apps
 */
public class RedCircle implements DrawAPI {
    @Override
    public void drawCircle(int radius, int x, int y) {
        System.out.println("Drawing Circle [color : red, radius: " + radius + ", x: " + x + ", y: " + y + "]");
    }
}

File: app/src/main/java/com/softaai/designpatterns/kotlin/structural/bridge/RedCircle.kt
package com.softaai.designpatterns.kotlin.structural.bridge


/**
 * Created by Amol Pawar on 03-04-2019.
 * softAai Apps
 */
class RedCircle : DrawAPI {
    override fun drawCircle(radius: Int, x: Int, y: Int) {
        println("Drawing circle[color : red, radius : " + radius + ", x : " + x + ", y : " + y + "]")
    }
}